sdks/python/scripts/run_integration_test.sh

```diff
@@ -48,6 +48,8 @@
 #                      during execution. Commonly used options like `--attr`,
 #                      `--tests`, `--nologcapture`. More can be found in
 #                      https://nose.readthedocs.io/en/latest/man.html#options
```

Comment:
`test_opts` is defined to hold Nose configs. Do you have particular reason to define it separately?

Reply:
Yes, it is also used in the xml output filename.